samples/CMakeLists.txt

```
@@ -21,3 +21,4 @@ add_subdirectory(rtc_timer)
 add_subdirectory(thermistor)
 add_subdirectory(timer)
 add_subdirectory(spi)
+add_subdirectory(encoder)
```

All files should end with a newline

Also, move this, so it's in alphabetical order

```suggestion
add_subdirectory(encoder)

```